com.civicrm.hrjobroles/js/src/job-roles/filters/option-values-filter.js

```diff
@@ -0,0 +1,22 @@
+define([
+  'common/lodash',
+  'job-roles/filters/filters'
+], function(_,filters) {
+  'use strict';
+
+  filters.filter('getActiveValues', ['$log', function($log) {
+    $log.debug('Filter: getActiveValues');
+
+    return function(optionValues) {
+      var filteredOptionValues = {};
+
+      _.each(optionValues, function (optionValue, idValue) {
+        if (optionValue.is_active === '1') {
+          filteredOptionValues[idValue] = optionValue;
+        }
+      });
+
+      return filteredOptionValues;
```

Comment:
Here in this case i need a `{}` but `_.filter(...)` return a `[]`. So, lodash filter can't be used.
Ref: https://jsfiddle.net/bmnepali/85tnzg7e/